osquery/core/sql/column.h

```diff
@@ -73,8 +73,10 @@ enum class ColumnOptions {
   /// This column should be hidden from '*'' selects.
   HIDDEN = 16,
 
-  // This sets the collating sequence to NOCASE
-  COLLATENOCASE = 32,
+  // Set the collation sequence for this column.
+  COLLATEBINARY = 32,
+  COLLATENOCASE = 64,
```

Comment:
Is renumbering this an issue?

Reply:
As far as I can tell it shouldn't. Should only change which bit position is used for that flag when enumerating the generated column options.